web/templates/pages/list_cron.php

```diff
@@ -188,10 +190,12 @@ class="units-table-row-action-link data-controls js-confirm-action"
 		<?php } ?>
 	</div>
 
-	<div class="units-table-footer">
+</div>
+
+<footer class="app-footer">
```

Comment:
Why have you added the global app footer in this template? This should already be included by `includes/app_footer.php`.